### PR TITLE
So wtd19 siteupdates

### PR DESCRIPTION
**Pull request by swapnilogale**

### CODE REVIEW

File: docs/_data/2019.australia.speakers.yaml [hunk 1]
```diff
@@ -117,63 +117,32 @@
     twitter: ''
     website: ''
   title: Rethinking Release Notes
-- abstract: '<p>We have been asked to do this very much lately. Customers don''t have
-    time or patience to sit down and read our User''s Guides, they would rather watch
-    a short "How to" video. </p>
+- abstract: '<p>Minimum Viable Product or MVP is a development technique where a
```
Comment (plaindocs) [inline docs/_data/2019.australia.speakers.yaml:120]: Is this right? Looks like it might be muddled.

Comment (swapnilogale) [reply docs/_data/2019.australia.speakers.yaml:120]: I just removed the video talk as the speaker cannot make it.

Comment (swapnilogale) [reply docs/_data/2019.australia.speakers.yaml:120]: @plaindocs - good for a merge now?

**Review by plaindocs — APPROVED** [review]
:+1:

**Pull request merged**